src/Microsoft.Framework.FileSystemGlobbing/Abstractions/DirectoryInfoWrapper.cs

```diff
@@ -39,9 +39,31 @@ public override IEnumerable<FileSystemInfoBase> EnumerateFileSystemInfos()
 
         public override DirectoryInfoBase GetDirectory(string name)
         {
-            return new DirectoryInfoWrapper(
-                new DirectoryInfo(Path.Combine(_directoryInfo.FullName, name)),
-                isParentPath: string.Equals(name, "..", StringComparison.Ordinal));
+            var isParentPath = string.Equals(name, "..", StringComparison.Ordinal);
```

Comment:
Just in case some alien culture makes into Unicode ... you know they may have uppercase `.`.  _I want to believe_